src/main/java/at/ac/tuwien/kr/alpha/solver/heuristics/BerkMin.java

```diff
@@ -50,29 +50,33 @@
 public class BerkMin implements BranchingHeuristic {
 	
 	public static final double DEFAULT_ACTIVITY = 0.0;
+	public static final int DEFAULT_SIGN_COUNTER = 0;
 	public static final int DEFAULT_CHOICE_ATOM = 0;
 
+	public static final int DEFAULT_DECAY_AGE = 10;
+	public static final double DEFAULT_DECAY_FACTOR = 0.25;
+
 	private Assignment assignment;
 	private Map<Integer, Double> activityCounters = new HashMap<>();
 	private Map<Integer, Integer> signCounters = new HashMap<>();
-	private Deque<NoGood> stackOfNoGoods = new ArrayDeque<NoGood>();
-	private int decayAge = 10;
-	private double decayFactor = 0.25;
+	private Deque<NoGood> stackOfNoGoods = new ArrayDeque<>();
+	private int decayAge;
+	private double decayFactor;
 	private int stepsSinceLastDecay;
 	private Random rand = new Random();
 	private Predicate<? super Integer> isAtomActiveChoicePoint;
 
-	public BerkMin(Assignment assignment, Predicate<? super Integer> isAtomActiveChoicePoint) {
+	public BerkMin(Assignment assignment, Predicate<? super Integer> isAtomActiveChoicePoint, int decayAge, double decayFactor) {
 		this.assignment = assignment;
 		this.isAtomActiveChoicePoint = isAtomActiveChoicePoint;
-	}
-
-	public BerkMin(Assignment assignment, Predicate<? super Integer> isAtomActiveChoicePoint, int decayAge, double decayFactor) {
-		this(assignment, isAtomActiveChoicePoint);
 		this.decayAge = decayAge;
 		this.decayFactor = decayFactor;
 	}
 
+	public BerkMin(Assignment assignment, Predicate<? super Integer> isAtomActiveChoicePoint) {
+		this(assignment, isAtomActiveChoicePoint, DEFAULT_DECAY_AGE, DEFAULT_DECAY_FACTOR);
```

Comment:
The defaulting constructor now does not do anything else except call `this`, which is simpler.